### PR TITLE
Add Webring Plugin

### PR DESCRIPTION
**Pull request by davidag**
This plugin retrieves the latest posts from a list of web feeds and makes them available in templates, effectively creating a [webring](https://en.wikipedia.org/wiki/Webring). Posts are sorted from newer to older.

It is inspired in [openring](https://git.sr.ht/~sircmpwn/openring), a tool for generating an html file to include in your Static Site Generator from a template and a list of web feeds.

This is an example of the plugin in use:

example

### CODE REVIEW

**Comment by davidag** [comment]
@Lucas-C I wasn't aware of the dedicated repositories policy that you just mentioned in #130, I only read the docs and this looked like the way to go :sob: 

In any case, it's perfectly OK for me to close this PR and follow the new guidelines when available :relaxed: 

Thank you and great work with the plugins repo! :tada:

**Comment by Lucas-C** [comment]
Hi @davidag !

There are 3 ways to create plugins:
- by adding a directory to this repo
- by creating a dedicated git repo and publishing it on Pypi
- and finally by creating a dedicated git repo and referencing it here as a submodule

I think the consensus is to avoid this last approach from now on.
However the other 2 options are totally fine for me. I personnally even prefer the "collective repo" approach as it potentially allow for more code reuse & code reviews.
As you want really :)

**Comment by justinmayer** [comment]
As I mentioned in my [related comment](https://github.com/getpelican/pelican-plugins/issues/130#issuecomment-529140401), I think the best approach for now is to continue with the current monolithic repository (option 1 above).

File: webring/README.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
+# Webring Plugin
+
+This plugin retrieves the latest posts from a list of web feeds and makes
+them available in templates, effectively creating a [partial webring]. Posts
```
Comment (justinmayer) [inline webring/README.md]: I think this was supposed to be a link, which presumably means it should instead be: `[partial webring][]`

File: webring/README.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+are sorted from newer to older.
+
+It is inspired in [openring](https://git.sr.ht/~sircmpwn/openring), a tool for
```
Comment (justinmayer) [inline webring/README.md]: This should instead be: "It is inspired _by_ [...]"

File: webring/README.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+generating an html file to include in your [SSG] from a template and a list of
```
Comment (justinmayer) [inline webring/README.md]: html → HTML. Also, to be a link, the `[SSG]` needs to be changed to `[SSG][]` (same issue as above).

File: webring/README.md (hunk 1 above)
Comment (davidag) [reply webring/README.md]: In fact it's a [reference link], specifically a `link label` :open_mouth: 

[reference link]: https://github.github.com/gfm/#reference-link "This is a reference link, too!"

File: webring/README.md (hunk 3 above)
Comment (davidag) [reply webring/README.md]: Fixed html lowercase. The link is also _reference link_.

File: webring/README.md (hunk 1 above)
Comment (justinmayer) [reply webring/README.md]: Hmm. Not familiar with that bit of (presumably) GitHub-specific Markdown syntax. Not sure I see the benefit of deviating from standard Markdown just to save two characters, but perhaps I'm misunderstanding something. 🤷‍♂️

Comment (davidag) [reply webring/README.md]: I was the one misunderstanding something :disappointed:

**Comment by justinmayer** [comment]
Nice work, David. Many thanks for the cool addition to the plugin ecosystem!

**Pull request merged**